src/SFA.DAS.ReferenceData.Organisations.Database/StoredProcedures/ImportDataFromLoadTables.sql

```diff
@@ -1,40 +1,28 @@
 ﻿CREATE PROCEDURE [CharityData].[ImportDataFromLoadTables]
 	WITH EXECUTE AS OWNER
 AS
-	IF OBJECT_ID('[CharityData].acct_submit', 'U') IS NOT NULL TRUNCATE TABLE [CharityData].acct_submit
-	IF OBJECT_ID('[CharityData].aoo_ref', 'U') IS NOT NULL TRUNCATE TABLE [CharityData].aoo_ref
-	IF OBJECT_ID('[CharityData].ar_submit', 'U') IS NOT NULL TRUNCATE TABLE [CharityData].ar_submit
+    IF OBJECT_ID('[CharityData].acct_submit', 'U') IS NOT NULL DROP TABLE [CharityData].acct_submit
+	IF OBJECT_ID('[CharityData].aoo_ref', 'U') IS NOT NULL DROP TABLE [CharityData].aoo_ref
+	IF OBJECT_ID('[CharityData].ar_submit', 'U') IS NOT NULL DROP TABLE [CharityData].ar_submit
+	IF OBJECT_ID('[CharityData].charity_aoo', 'U') IS NOT NULL DROP TABLE [CharityData].charity_aoo
+	IF OBJECT_ID('[CharityData].class', 'U') IS NOT NULL DROP TABLE [CharityData].class
+	IF OBJECT_ID('[CharityData].class_ref', 'U') IS NOT NULL DROP TABLE [CharityData].class_ref
+	IF OBJECT_ID('[CharityData].financial', 'U') IS NOT NULL DROP TABLE [CharityData].financial
+	IF OBJECT_ID('[CharityData].name', 'U') IS NOT NULL DROP TABLE [CharityData].name
+	IF OBJECT_ID('[CharityData].[objects]', 'U') IS NOT NULL DROP TABLE [CharityData].[objects]
+	IF OBJECT_ID('[CharityData].overseas_expend', 'U') IS NOT NULL DROP TABLE [CharityData].overseas_expend
+	IF OBJECT_ID('[CharityData].partb', 'U') IS NOT NULL DROP TABLE [CharityData].partb
+	IF OBJECT_ID('[CharityData].remove_ref', 'U') IS NOT NULL DROP TABLE [CharityData].remove_ref
+	IF OBJECT_ID('[CharityData].trustee', 'U') IS NOT NULL DROP TABLE [CharityData].trustee
```

Comment:
These should not be needed if we have removed the tables from the dacpac. They will be removed on the next deployment anyway